src/core/server/rendering/bootstrap/bootstrap.js.template.hbs

```diff
@@ -1,8 +1,37 @@
+function kbnBundlesLoader() {
+  var modules = {};
+
+  function has(prop) {
+    return Object.prototype.hasOwnProperty.call(modules, prop);
```

Comment:
The logic to serialize and 'render as string' the `kbnBundlesLoader` implementation felt overcomplicated and unnecessary. I inlined it in the `bootstrap.js` template instead.